sql-cli/sql_cli/connections.py

```diff
@@ -58,17 +60,16 @@ def validate_connections(environment: str = "default", connection_id: str | None
         if connection_id:
             config_file_contains_connection = True
         conn_obj = Connection(**data)
-
         _create_or_replace_connection(conn_obj)
 
         success_status, _ = conn_obj.test_connection()
```

Comment:
This could be rewritten to use the local backend.

It is confusing that we rely on the `airflow.db` here while the `run` command uses the local backend. Ideally, we'd be consistent.